EIPS/eip-2315.md

```diff
@@ -2,7 +2,7 @@
 eip: 2315
 title: Simple Subroutines for the EVM
 description: This proposal introduces two opcodes to support simple subroutines.
```

Comment:
I think the "this proposal" can be dropped as it can be inferred by the context of the description.